infra/Pos/Discovery/Holders.hs

```diff
@@ -0,0 +1,79 @@
+{-# LANGUAGE DataKinds #-}
+-- | Transformer that carries peer discovery capabilities.
+
+module Pos.Discovery.Holders
+       ( DiscoveryTag
+       , DiscoveryConstT
+       , runDiscoveryConstT
+       , DiscoveryKademliaT
+       , askDHTInstance
+       , runDiscoveryKademliaT
+       ) where
+
+import qualified Control.Monad.Ether              as Ether
+import qualified Control.Monad.Ether              as Ether.E
+import qualified Data.Set                         as S (fromList)
+import           Mockable                         (Async, Catch, Mockables, Promise,
+                                                   Throw)
+import           System.Wlog                      (WithLogger)
+import           Universum
```

Comment:
But anyway, why don't you put `Universum` as the first import? I don't insist on fixing it now, but we agreed to do it long ago :/